### PR TITLE
PPTP-1019 Reorder Liability Details on Registration page

### PR DESCRIPTION
**Pull request by PauloPortugal**
This also meant enabling the link for that section, as it will be the
first section the user will see in the new PPT prototype version.

### CODE REVIEW

File: app/uk/gov/hmrc/plasticpackagingtax/registration/models/registration/Registration.scala [hunk 1]
```diff
@@ -73,7 +73,7 @@ case class Registration(
 
   def liabilityDetailsStatus: TaskStatus =
     if (incorpJourneyId.isEmpty)
-      TaskStatus.CannotStartYet
+      TaskStatus.NotStarted
```
Comment (RyanKellyHMRC) [inline app/uk/gov/hmrc/plasticpackagingtax/registration/models/registration/Registration.scala:76]: Is this the right check? Surely we should be looking to see whether we have recorded any liability details in the registration?

**Review by RyanKellyHMRC — CHANGES_REQUESTED** [review]

File: test/uk/gov/hmrc/plasticpackagingtax/registration/models/registration/RegistrationSpec.scala [hunk 2]
```diff
@@ -55,10 +55,10 @@ class RegistrationSpec
         notStartedRegistration.numberOfCompletedSections mustBe 0
 
         notStartedRegistration.isCompanyDetailsComplete mustBe false
-        notStartedRegistration.companyDetailsStatus mustBe TaskStatus.NotStarted
+        notStartedRegistration.companyDetailsStatus mustBe TaskStatus.CannotStartYet
 
         notStartedRegistration.isLiabilityDetailsComplete mustBe false
-        notStartedRegistration.liabilityDetailsStatus mustBe TaskStatus.CannotStartYet
+        notStartedRegistration.liabilityDetailsStatus mustBe TaskStatus.NotStarted
```
Comment (RyanKellyHMRC) [inline test/uk/gov/hmrc/plasticpackagingtax/registration/models/registration/RegistrationSpec.scala:61]: It probably makes sense to reorder these checks to reflect the on-page order.

**Review by RyanKellyHMRC — APPROVED** [review]

File: test/uk/gov/hmrc/plasticpackagingtax/registration/models/registration/RegistrationSpec.scala (hunk 2 above)
Comment (PauloPortugal) [reply test/uk/gov/hmrc/plasticpackagingtax/registration/models/registration/RegistrationSpec.scala:61]: Done!

Comment (RyanKellyHMRC) [reply test/uk/gov/hmrc/plasticpackagingtax/registration/models/registration/RegistrationSpec.scala:58]: I actually meant the companyDetails, liabilityDetails checks should be made in the same order as the sections on the page but don't worry it should not hold up a merge.

**Pull request merged**